src/runtime/runtime.ml

```diff
@@ -683,10 +685,10 @@ let print_external ?max_level ~penv v ppf =
 (** In the future this routine will be type-driven. One consequence is that
     constructor tags will be printed by looking up their names in type
     definitions. *)
-let rec print_value ?max_level ~penv v ppf =
+let rec print_value ?max_level ~penv ~sgn v ppf =
   match v with
 
-  | Judgement jdg -> Nucleus.print_judgement_abstraction ~penv:(mk_nucleus_penv penv) ?max_level jdg ppf
+  | Judgement jdg -> Nucleus.print_judgement_abstraction ~penv:(mk_nucleus_penv penv) ?max_level ~sgn jdg ppf
```

Comment:
Do it in two steps: first compute the judgement with boundary from `jdg` and `sgn`, then call `Nucleus.print_judgement_with_boundary_abstrction` (which therefore does not need ~sgn`).